README.md

```diff
@@ -34,20 +34,16 @@ We are a Cloud Native Computing Foundation (CNCF) sandbox project.
 - [Getting started](#getting-started)
 - [Deploying KEDA](#deploying-keda)
 - [Documentation](#documentation)
+  - [Become a listed KEDA user!](#become-a-listed-keda-user)
```

Comment:
bad intendation

Reply:
Generated by this doctoc tool 🤷‍♂️ Happy to change if you want.

Reply:
Using `##` instead of `###` for `Become a listed KEDA user!` section will fix the issue 👍